onnx/cppbuild.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# This file is meant to be included by the parent cppbuild.sh script
+if [[ -z "$PLATFORM" ]]; then
+    pushd ..
+    bash cppbuild.sh "$@" onnx
+    popd
+    exit
+fi
+
+if [[ $PLATFORM == windows* ]]; then
+    #No Windows support yet
+    :
+else
+
+    export PROTO=2.6.1
+    export INSTALL_PATH=`pwd`/$PLATFORM
+    wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O ~/miniconda.sh
+    bash ~/miniconda.sh -f -b -p $HOME/miniconda 
```

Comment:
Does Miniconda need to be installed in the $HOME directory? What happens if we place it in $INSTALL_PATH?